app/User.php

```diff
@@ -1,38 +1,26 @@
 <?php
 
+declare(strict_types=1);
+
 namespace App;
 
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
+use Laravel\Sanctum\HasApiTokens;
 
 class User extends Authenticatable
 {
-    use Notifiable;
+    use HasApiTokens, Notifiable;
```

Comment:
Please divide trait injections according to PSR-12: https://www.php-fig.org/psr/psr-12/#42-using-traits